src/tools/rbd_mirror/ImageReplayer.h

```diff
@@ -195,6 +196,35 @@ class ImageReplayer {
     ImageReplayer<ImageCtxT> *replayer;
   };
 
+  struct ReplayStatusMaker {
```

Comment:
Actually it accesses private m_remote_journaler and m_local_mirror_uuid. Sure it can be resolved, so I will move it out into the "image_replayer".

I don't like "Maker" very much too, will change.